packages/rum-core/src/browser/cookieObservable.ts

```diff
@@ -0,0 +1,68 @@
+import type { Configuration, CookieStore } from '@datadog/browser-core'
+import {
+  setInterval,
+  clearInterval,
+  Observable,
+  addEventListener,
+  ONE_SECOND,
+  findCommaSeparatedValue,
+  DOM_EVENT,
+} from '@datadog/browser-core'
+
+export interface CookieStoreWindow extends Window {
+  cookieStore?: CookieStore
+}
+
+export type CookieObservable = ReturnType<typeof createCookieObservable>
+
+export type CookieChange = { name: string; value: string | undefined }
+
+export function createCookieObservable(configuration: Configuration, cookieName: string) {
```

Comment:
**💬 suggestion:** ‏Since the `CookieObservable` is scoped to a single cookie name, it could simply notify the value (`string|undefined`) instead of an object with its `name`